docs/notebooks/failure_ego.pct.py

```diff
@@ -48,9 +51,7 @@
 
 # %%
 def masked_branin(x):
-    x0 = np.abs(x[:, 0] - 0.5) < 0.25
-    x1 = np.abs(x[:, 1] - 0.5) < 0.40
-    mask_nan = np.logical_and(x0, x1)
+    mask_nan = np.sqrt((x[:, 0] - 0.5) ** 2 + (x[:, 1] - .4) ** 2) < 0.3
```

Comment:
you'll need to update the text above to account for the change in shape of the failure region